src/Products/Viewer/Controllers/ViewerApiController.cs

```diff
@@ -203,42 +217,52 @@ public HttpResponseMessage RotateDocumentPages(PostedDataEntity postedData)
         {
             try
             {
-                // get/set parameters
-                string documentGuid = postedData.guid;
-                int angle = postedData.angle;
-                List<int> pages = postedData.pages;
-                string password = postedData.password;
-                DocumentInfoOptions documentInfoOptions = new DocumentInfoOptions(documentGuid);
-                // set password for protected document                
-                documentInfoOptions.Password = password;
-                // a list of the rotated pages info
-                List<RotatedPageEntity> rotatedPages = new List<RotatedPageEntity>();
-                // rotate pages
-                for (int i = 0; i < pages.Count; i++)
+                var documentGuid = postedData.guid;
+                var pageNumber = postedData.pages[0];
+
+                string pageFilePathFormat;
+                ViewerSettings settings = GetViewerSettings(documentGuid, out pageFilePathFormat);
+
+                var cacheFolder = ((FileCache)settings.Cache).CachePath;
+                File.Delete(Path.Combine(cacheFolder, $"p{pageNumber}_html.dat"));
+                File.Delete(Path.Combine(cacheFolder, $"p{pageNumber}_png.dat"));
+
+                var page = new PageDescriptionEntity();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this useless assignment to local variable 'page'.](https://app.codacy.com/manual/GroupDocs/GroupDocs.Viewer-for-.NET-MVC/pullRequest?prid=5242613)